lib/hanami/web/welcome.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require "erb"
```

Comment:
do we now need a runtime dependency for erb or are we assuming it'll continue to ship with the standard library (this is true for upcoming version 3.3 so probably fine for another year at least!)

Reply:
Good question! Personally, I'm comfortable relying on erb as distributed with Ruby for as long as we can.